service/resource/s3bucketv1/resource.go

```diff
@@ -0,0 +1,90 @@
+package s3bucketv1
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/framework"
+
+	awsutil "github.com/giantswarm/aws-operator/client/aws"
+	awsservice "github.com/giantswarm/aws-operator/service/aws"
+)
+
+const (
+	// Name is the identifier of the resource.
+	Name = "s3bucket"
+)
+
+// Config represents the configuration used to create a new s3bucket resource.
+type Config struct {
+	// Dependencies.
+	AwsService *awsservice.Service
+	Clients    Clients
+	Logger     micrologger.Logger
+
+	// Settings.
+	AwsConfig awsutil.Config
```

Comment:
Do we need the AwsConfig here?

Reply:
No I'd missed removing it. Thanks its gone now.